### DISCUSSION TITLE
Tutorial - User guide - First steps uses two different examples

### DISCUSSION THREAD

**Discussion by florisje** [fastapi/fastapi#8683] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
It's a documentation issue, no code.
```

### Description

The First Steps page in the Tutorial - User guide starts with using the Hello World example.

But then at the Interactive API docs step it uses the Items example.

This is confusing.

### Operating System

macOS

### FastAPI Version

0.70.0

### Python Version

Python 3.8.11

**Comment by jrwren** [comment]
related: tutorial/first-steps/#interactive-api-docs the screenshot does not match the code at that point in time. This is confusing. /items/{item_id} is not expected docs for

```
@app.get("/")
async def root():
    return {"message": "Hello World"}
```
